deploy/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -1,17 +1,7 @@
 # RUNTIME configuration for the FireSim Simulation Manager
 # See https://docs.fires.im/en/stable/Advanced-Usage/Manager/Manager-Configuration-Files.html for documentation of all of these params.
 
-run_farm:
-    run_farm_tag: mainrunfarm
-    always_expand_runfarm: yes
-    f1_16xlarges: 1
-    m4_16xlarges: 0
-    f1_4xlarges: 0
-    f1_2xlarges: 0
-    launch_instances_timeout_minutes: 60
-    run_instance_market: ondemand
-    spot_interruption_behavior: terminate
-    spot_max_price: ondemand
+run_farm: aws_ec2_f1_run_farm
```

Comment:
```
run_farm:
  run_farm_recipe: <PATH TO DEFAULTS FILE>
  overrride_args:
     <ARG> : <OVERRIDE>
```

Reply:
Supposedly its easy to combine the two different dicts (w/ nesting taken into account).

Reply:
managerinit will copy the default args of the platform given into the override section and point the runfarmrecipe to that default file. in the workloads yaml we will just point to a f1 recipe by default (in a file).


instance counts are defaulted to 0 (or whatever it was b4)

Reply:
Implemented. Closing